sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/webpack-stats/test

```diff
@@ -0,0 +1 @@
+> fullOptJS::webpack
```

Comment:
I don't understand how this test relates to the fix. If this PR is about failing a build, there should a be at least one negative test line (starting with `-`).

Reply:
This test would fail in current `main` branch, the changes added introduce fixes that actually make it pass. Is that fine?